VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -170,13 +170,9 @@ def exit_if_same_seq(self, exit_status=None):
             sys.exit(0)
 
     def log(self, message):
-        # write message to stderr for inclusion in QOS telemetry
-        sys.stderr.write(message)
         self._log(self._get_log_prefix() + ': ' + message)
 
     def error(self, message):
-        # write message to stderr for inclusion in QOS telemetry
-        sys.stderr.write(message)
```

Comment:
@vimish why did we do this again? been to long; I think it was a bug you may have pointed to fix while we are updating